components/nodemanager-service/pgdb/filtering.go

```diff
@@ -38,31 +38,31 @@ func mergeFilters(mergeableFilters []*common.Filter) ([]common.Filter, error) {
 	return filters, nil
 }
 
-func handleTagFilters(tagFilters []common.Filter) (string, error) {
-	var tagConditions []string
+func handleTagFilters(tagFilters []common.Filter) ([]string, error) {
```

Comment:
I'm wondering if this is a clearer function name we might use here.  `TagFiltersToSQLConditions` maybe?